### DISCUSSION TITLE
Sub Organization Feature Request

### DISCUSSION THREAD

**Discussion by kroger-ktomazin** [community/community#5627] [Other Feature Feedback, Questions, & Ideas]
Hi, I'd like to request that you add the ability to create a hierarchy of Organizations.  I work for a large company and would like to create a single set of secrets for all the repos which I manage but the only way I see to do that is to create them at the Organization level and then specify which repos they apply to.  Since I am not (nor should I be) an admin at the Organization level, I would have to make a request to another group to create my secrets and then update the list of repositories to which they apply each and every time I create a new repo.  Since this would be a bureaucratic and maintenance nightmare, the only other option I have to redundantly create the same secrets for each and every repository which obviously violates the DRY principle.  If however, there were an easy (and free) way of creating a sub-organization, then I could create my repos and secrets at that level and I would be all set.

**Comment by dclayton-godaddy** [comment]
Would love to see this for secrets, runners, and repo defaults.
